projects/nginx/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google Inc.
+# Modifications copyright (C) 2024 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+# Install build dependencies.
+RUN apt-get install -y build-essential libssl-dev
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.30.1/cmake-3.30.1.tar.gz && \
```

Comment:
Do we need to install this current version of cmake? Apt version doesn't work?

Reply:
Apt version doesn't fit for building libprotobuf